src/modules/personenkontext/api/param/dbiam-update-personenkontexte.query.params.ts

```diff
@@ -0,0 +1,10 @@
+import { ApiProperty } from '@nestjs/swagger';
+
+import { IsOptional, IsString } from 'class-validator';
+
+export class DbiamUpdatePersonenkontexteQueryParams {
+    @IsString()
+    @IsOptional()
+    @ApiProperty({ required: false })
+    public readonly personalnummer?: string;
+}
```

Comment:
Die Personalnummer sollte in die Bodyparams als optionaler Parameter, ggf. mit einer Referenz auf den bezogenen Kontext wegen dem Sie mitgesendet wird.